cosmo_tester/resources/scripts/snapshots_migration_scripts/bootstrap_manager_3_2_1.template

```diff
@@ -0,0 +1,15 @@
+set -eax
+orig_dir=$(pwd)
+cd {{ work_dir }}
+source {{ venv_name }}/bin/activate >/dev/null
+
+cd {{ repo_path }}
+git checkout 3.2.1-build
+cp {{ work_dir }}/{{ updated_blueprint }} ./openstack/
+cd {{ work_dir }}/{{ venv_name }}
+cfy bootstrap --install-plugins -i {{ work_dir }}/{{ inputs_file }} -p {{ repo_path }}/openstack/{{ updated_blueprint }}
+
+sleep 10
```

Comment:
Why sleep here? If manager is not "really ready" straight after bootstrap, I would perform some sleep-loop until, let's say, cfy status returns good news or I don't know, cfy deployments list exits with 0..